### PR TITLE
HTBHF-1973 Updated content of the unsuccessful application page

### PR DESCRIPTION
**Pull request by YetAnotherMatt**

### CODE REVIEW

File: src/web/views/guidance/en/what-you-get.njk [hunk 1]
```diff
@@ -55,7 +55,7 @@
 
 <h3 class="govuk-heading-m">Vitamin vouchers</h3>
 
-<p class="govuk-body">Vitamins are not part of the card service yet. You’ll be sent vouchers for those separately which you can exchange for vitamins.</p>
+<p class="govuk-body">Vitamins are not part of the card service yet. You’ll be sent vouchers for them separately if you want them. You can exchange them for vitamins.</p>
```
Comment (YetAnotherMatt) [inline src/web/views/guidance/en/what-you-get.njk]: This is an unrelated change arising from the review of HTBHF-1910

File: src/web/views/locales/en/unsuccessful/error.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<p class="govuk-body">An error occurred whilst checking your eligibility</p>
+<p class="govuk-body"> <a href="apply">Go back and try again</a>.</p>
```
Comment (dwybourn) [inline src/web/views/locales/en/unsuccessful/error.html]: Should the links have the govuk-link class attached? Or, should we finally update the css so we don't have to keep doing this.

File: src/web/views/locales/en/unsuccessful/pending.html [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<p class="govuk-body">Your benefit status is pending</p>
+<p class="govuk-body">If you’re still having problems call 0300 330 7010 and select option 3.</p>
```
Comment (YetAnotherMatt) [inline src/web/views/locales/en/unsuccessful/pending.html]: Pending content hasn't been provided yet - I just added the phone number to the bottom of all decisions.

File: src/web/views/locales/en/unsuccessful/error.html (hunk 2 above)
Comment (mikeSmith08) [inline src/web/views/locales/en/unsuccessful/error.html]: You've missed the class from the <a> tag.
